### PR TITLE
Add --config option for arbitrary config file

### PR DESCRIPTION
**Pull request by fflorens**
Simple commit to add the ability to use an arbitrary config file instead of `~/.scwrc`

### CODE REVIEW

**Comment by sieben** [comment]
LGTM

**Comment by brmzkw** [comment]
It would be great to add an unittest

File: pkg/cli/main.go [hunk 1]
```diff
@@ -47,6 +48,10 @@ func Start(rawArgs []string, streams *commands.Streams) (int, error) {
 	}
 	flag.CommandLine.Parse(rawArgs)
 
+	if *flConfig != "" {
+		os.Setenv("SCW_CONFIG_PATH", *flConfig)
```
Comment (brmzkw) [inline pkg/cli/main.go]: I think you should not set the environment variable here, but instead return a default if `SCW_CONFIG_PATH` is not set below in `pkg/config/config.go`.

**Comment by fflorens** [comment]
Rebased against master

**Comment by brmzkw** [comment]
thanks! <3

**Pull request merged**